godot-macros/src/derive_godot_class.rs

```diff
@@ -321,3 +418,32 @@ fn make_exports_impl(class_name: &Ident, fields: &Fields) -> TokenStream {
         }
     }
 }
+
+/// Checks at compile time that a function with the given name exists on `Self`.
+#[must_use]
+fn existence_check(ident: &Ident) -> TokenStream {
+    quote! {
+        #[allow(path_statements)]
+        Self::#ident;
+    }
+}
```

Comment:
Maybe name it `make_existence_check` -- this currently sounds like it checks for the existence, not like it _generates_ code that does so.

Reply:
Usually functions named like a noun are functions in the mathematical sense, which compute something based on their arguments and return a value. E.g. `cos(x)` computes and returns the cosine of `x`; we don't call this `compute_cos`. A function that checks for the existence should be named `check_existence` instead.

But I'll rename this one anyway for consistency with its surroundings :)

Reply:
One problem with English is that so many words are simultaneously nouns and verbs. There are many functions/macros which are named as `prefix_action`, e.g. `godot_print`, `static_assert`, `variant_call`, etc.

So I think `make_`, apart from being consistent, helps resolve this ambiguity.